tailwind.config.js

```diff
@@ -14,155 +14,179 @@ module.exports = {
+      h3: ['1.875rem', '3rem'],
+      h4: ['1.625rem', '2.5rem'],
+      h5: ['1.25rem', '2rem'],
+      lead: ['1.5rem', '2.25rem']
+    },
+    spacing: {
+      // The px values for spacing are equivalent to their keys.
+      px: '1px',
+      0: '0',
+      4: '0.25rem',
+      8: '0.5rem',
+      12: '0.75rem',
+      16: '1rem',
+      20: '1.25rem',
+      24: '1.5rem',
+      28: '1.75rem',
+      32: '2rem',
+      36: '2.25rem',
+      40: '2.5rem',
+      44: '2.75rem',
+      48: '3rem',
+      52: '3.25rem',
+      56: '3.5rem',
+      60: '3.75rem',
+      64: '4rem',
+      68: '4.25rem',
+      72: '4.5rem',
+      76: '4.75rem',
+      80: '5rem'
+    },
+    borderWidth: {
+      DEFAULT: '0.125rem'
+    },
+    borderRadius: {
+      none: '0',
+      DEFAULT: '0.25rem',
+      larger: '0.375rem',
+      largest: '0.5rem',
+      full: '9999px'
+    },
+    colors: {
+      primary: {
+        lightest: '#FCD0C4',
+        lighter: '#F9A189',
+        light: '#F78161',
+        DEFAULT: '#F5623A',
+        dark: '#DD5834',
+        darker: '#AC4529',
+        darkest: '#622717'
+      },
+      secondary: {
+        lightest: '#D2E9FA',
+        lighter: '#A5D4F5',
+        light: '#62B3EE',
+        DEFAULT: '#1F93E7',
+        dark: '#1976B9',
+        darker: '#104A74',
+        darkest: '#092C45'
+      },
+      tertiary: {
+        lightest: '#ECECEC',
+        lighter: '#C6C6C6',
+        light: '#7A7A7A',
+        DEFAULT: '#414141',
+        dark: '#343434',
+        darker: '#272727',
+        darkest: '#1A1A1A'
+      },
+      success: {
+        lightest: '#D1EEDD',
+        lighter: '#A3DDBC',
+        light: '#5DC389',
+        DEFAULT: '#18A957',
+        dark: '#11763D',
+        darker: '#0A4423',
+        darkest: '#052211'
+      },
+      warning: {
+        lightest: '#FFF1D7',
+        lighter: '#FFE4AF',
+        light: '#FFCF74',
+        DEFAULT: '#FFBB38',
+        dark: '#B38327',
+        darker: '#664B16',
+        darkest: '#33250B'
+      },
+      error: {
+        lightest: '#F9D0D9',
+        lighter: '#F2A2B3',
+        light: '#E95C7B',
+        DEFAULT: '#DF1642',
+        dark: '#9C0F2E',
+        darker: '#2D040D',
+        darkest: '#2D040D'
       },
-      fontFamily: {
-        sans: ['Libre Franklin', 'sans-serif'],
-        serif: ['Libre Baskerville', 'serif']
+      grey: {
+        lightest: '#FCFCFC',
+        lighter: '#E6E6E6',
+        light: '#939393',
+        DEFAULT: '#666666',
+        dark: '#5C5C5C',
+        darker: '#333333',
+        darkest: '#212121'
       },
-      fontSize: {
-        'root-em': '16px',
-        uber: '7rem',
-        hero: '5rem',
-        body: '1.125rem',
-        sm: '1rem',
-        caption: '0.875rem',
-        xs: '0.75rem',
-        xxs: '0.625rem',
-        h1: '3rem',
-        h2: '2.25rem',
-        h3: '1.875rem',
-        h4: '1.625rem',
-        h5: '1.25rem',
-        lead: '1.5rem'
+      white: {
```

Comment:
See me on this, I saw some tailwind syntax that might make this unnecessary. Instead of defining all of our opacities could just define the color ex: `bg-black` and bg-opacity-10`. I believe this works on all types of coloring.

Reply:
Just adding some insight here, I'm referring to the color `opacities` that are defined in the tailwind config. I realize the selected code does not show that.

Reply:
Good idea Mike! It was silly not to use tailwind's opacity utility classes for this. I changed it eaada5035223b32ca3447115031e23b0913f474f.